### ISSUE TITLE
Finetuning stage

### ISSUE THREAD

**Issue by seongwn** [jxhuang0508/CaCo#7]
Hello, thanks for your great work!

It seems that the objective function in finetuning stage includes 1) segmentation loss in the target domain (Cross-entropy) 2) Adversarial learning 3) Contrastive learning loss.

Also, it seems that you only use the target data (Cityscapes) in the finetuning stage.

Why did you not use source data in the finetuning stage, and did the adversarial training work well despite the absence of the source dataset?

Thank you very much.
